src/client/sandbox/shadow-ui.js

```diff
@@ -269,15 +271,37 @@ export default class ShadowUI extends SandboxBase {
         }
     }
 
+    overrideElement (el) {
+        if (domUtils.isBodyElement(el) || domUtils.isHeadElement(el)) {
```

Comment:
We already overrode `head` and `body` elements on prototype level. see above.
`overrideElement` method should be removed.